pkg/inputs/mysqlbatch/position_value.go

```diff
@@ -302,7 +312,42 @@ func PutCurrentPos(cache position_store.PositionCacheInterface, fullTableName st
 		return errors.Trace(err)
 	}
 
+	// update current position and increment scan count
 	batchPositions.Current[fullTableName] = *pos
+	stats := batchPositions.Stats[fullTableName]
+	stats.ScannedCount++
```

Comment:
put this into after commit hook is better I think. `PutCurrentPos` might be called for other purpose.